src/modules/database/mssql-ods-database.module.ts

```diff
@@ -0,0 +1,37 @@
+import { Module } from '@nestjs/common';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { DATABASE } from '@ukef/constants';
+
+@Module({
+  imports: [
+    TypeOrmModule.forRootAsync({
+      name: DATABASE.ODS,
+      imports: [ConfigModule],
+      inject: [ConfigService],
+      useFactory: (configService: ConfigService) => ({
+        name: DATABASE.ODS,
+        host: configService.get<string>('database.mssql_ods.host'),
+        port: configService.get<number>('database.mssql_ods.port'),
+        database: configService.get<string>('database.mssql_ods.name'),
+        type: 'mssql',
```

Comment:
We have this repeated quite often, can you please convert this to a constant?

Reply:
I've extracted it, a lot of files are now changed due to renaming constants

Reply:
Thank you